Anlab.Mvc/Views/Lab/UpdateFromCompletedTests.cshtml

```diff
@@ -0,0 +1,35 @@
+﻿@using Humanizer
+@model AnlabMvc.Models.Order.OrderReviewModel
+
+@{
+    ViewData["Title"] = "Confirmation";
+}
+
+@Html.Partial("_OrderDetails")
+
+<form asp-controller="Lab" asp-action="UpdateFromCompletedTests" asp-route-id="@Model.Order.Id" method="post" class="form-horizontal">
+    @Html.Hidden("confirm", true)
+    <button type="submit" class="btn btn-primary"><i class="fa fa-check" aria-hidden="true"> Confirm Completed (Update Tests from Labworks)</i></button>    
+</form>
+
+
+<div class="form-group">
+    <strong>json Data</strong>
+    <span>@Model.Order.JsonDetails</span>
+</div>
+
+@section AdditionalStyles
+{
+    <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.13/css/jquery.dataTables.min.css" />
+}
+
+@section Scripts
+{
+    <script type="text/javascript" src="https://cdn.datatables.net/1.10.13/js/jquery.dataTables.min.js"></script>
```

Comment:
@jSylvestre has nothing to do with this PR but it reminded me -- go ahead and create a set of partials for datatables, so you can do like `@Html.Partial("_DataTablesScripts")`.  Or if you have another idea that's good too, my goal is not to repeat the CDN path more than once, I'd hate to upgrade and have to change in each file.